app-ios/Sources/SearchFeature/FilterButtonView.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+import Theme
+
+struct FilterButtonView: View {
```

Comment:
I think this view has some common points with `FavoriteToggleButtonView`.
Please extract this view as CustomView and use in both view 🙏🏼